lib/widgets/payment_request_dialog.dart

```diff
@@ -36,6 +37,10 @@ class PaymentRequestDialogState extends State<PaymentRequestDialog>
     with SingleTickerProviderStateMixin {
 
   final _formKey = GlobalKey<FormState>();
+  GlobalKey _titleKey = GlobalKey();
```

Comment:
Since you are using fixed heights for title and actions, why do we need the `_titleKey` and `_actionsKey` ?

Reply:
There are cases where there are no title or action widgets rendered.

I will calculate the height if they exist and they will take precedence over the constant values.